### ISSUE TITLE
How to get selected values?

### ISSUE THREAD

**Issue by tschopp-spear** [NielsMinnee/NNDropdown#12]
Is there a way to get the selected values in a javascript save event handler method?  I'm trying this command and it's returning null:

var selectedVals = formContext.getAttribute("xyz_multiselectfield").getValue();

**Comment by NielsMinnee** [comment]
Hi, the select/deselect is instantly creating/deletion the N:N association. You should query the N:N table. Try using the FetchXML builder in the XRM Toolbox.

Message can be short, sent from my mobile

Kind regards, met vriendelijke groet,

Niels Minnee
Mobile: +31 6 50 60 66 75<tel:+31%206%2050%2060%2066%2075> || Office: +31 20 30 32 47 0 <tel:+31%2020%2030%2032%2047%200>

[cid:frame14nowhitespace_9989cfb9-a06f-4680-8226-194f67b0bfdb.png]
www.dynamicpeople.com <http://www.dynamicpeople.com/>  ||  www.be-empowered.nl<http://www.be-empowered.nl/>  ||  Schipluidenlaan 4 || 1062 HE || Amsterdam || The Netherlands

The information contained in this communication is confidential and intended solely for the use of the individual or entity to whom it is addressed. You should not copy, disclose or distribute this communication without the authority of Dynamic People B.V. Dynamic People is neither liable for the proper and complete transmission of the information contained in this communication nor for any delay in its receipt. Dynamic People does not guarantee that the integrity of this communication has been maintained nor that the communication is free of viruses, interceptions or interference. If you are not the intended recipient of this communication, please return the communication to the sender and delete and destroy all copies.




________________________________
From: tschopp-spear ***@***.***>
Sent: Monday, June 13, 2022 9:04:36 PM
To: NielsMinnee/NNDropdown ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [NielsMinnee/NNDropdown] How to get selected values? (Issue #12)

External Email

Is there a way to get the selected values in a javascript save event handler method? I'm trying this command and it's returning null:

var selectedVals = formContext.getAttribute("xyz_multiselectfield").getValue();

**Comment by tschopp-spear** [comment]
Got it, thanks!  It looks like this control can't be used when adding a new record and the ID isn't established yet.  Is that correct?

**Comment by NielsMinnee** [comment]
That's true because it's an association between two id's. However in a different implementation it should be possible by using an 'after-save' event.
